IORawData/HcalTBInputService/plugins/HcalTBSource.cc

```diff
@@ -21,7 +21,10 @@ HcalTBSource::HcalTBSource(const edm::ParameterSet& pset, edm::InputSourceDescri
   m_tree = nullptr;
   m_fileCounter = -1;
   m_file = nullptr;
-  m_i = 0;
+  m_i = m_skip;
+
+  if (m_skip != 0)
+    edm::LogWarning("HcalTBSource: skipEvents != 0 works only for the first input file ");
```

Comment:
```suggestion
    edm::LogWarning("HcalTBSource") << "skipEvents != 0 works only for the first input file";
```
haha

Reply:
You only dealt with half of what I suggested earlier :)

Reply:
@abdoulline also, can you please squash these 3 commits now into 1? 3 commits changing 1 file is not great :) Thanks